pkgs/build-support/rust/cargo-local-registry.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchurl, stdenv }:
+
+let
+  version = "0.1.4";
+  platform =
+    if stdenv.system == "x86_64-linux"
```

Comment:
We will also need `aarch64`. Maybe we can work-around getting cargo build in bootstrap phase without this package and build `cargo-local-registry` from source.

Reply:
@Mic92 As far as i can tell there is currently no support for rust bootstrap for aarch64 either? There are aarch64 builds available on the official [rust installer page](https://www.rust-lang.org/en-US/other-installers.html). Adding a binaries for aarch64 to upstream `cargo-local-registry` would be easy tough.

@kevincox I was working on something similar but based on [cargo-vendor](https://github.com/alexcrichton/cargo-vendor).  Not sure yet which one is better suited but according to the authore (which is the same for both) `cargo-vendor` is getting more attention at the moment.

Reply:
@kevincox I forget to mention that `cargo-vendor` might be easier to bootstrap soon as it will soon have a tarball with vendored dependencies available. See: https://github.com/alexcrichton/cargo-vendor/pull/51

Reply:
So I'm not an expert on the differences but it seems like cargo-local-registry is designed to "mirror" the required dependencies whereas cargo-vendor is supposed to the upstream project to vendor dependencies themselves. Either will probably work in practice but cargo-local-registry looks like a better fit for our use case.

And it seems like the bootstrapping issue shouldn't be a major concern as cargo can cross-compile easily. Someone just needs to but in some effort to bootstrap the first version.

That being said I'm happy for whatever solution works.